addons/smtp-mail.sh

```diff
@@ -224,7 +224,21 @@ set sendmail="/usr/bin/msmtp -t"
 DEFINE_MAIL
 
 # Test sending of mails
-if ! echo -e "Congratulations! Since this email reached you, it seems like everything is working properly. :) To change the settings please check /etc/msmtp on your server, or simply just run the setup script again." | mail -s "Test email from your NcVM" "$RECIPIENT" &>/dev/null
+if ! echo -e "Congratulations!
+
+Since this email reached you, it seems like everything is working properly. :)
+
+To change the settings please check /etc/msmtprc on your server, or simply just run the setup script again.
+
+YOUR CURRENT SETTINGS:
+-------------------------------------------
+$(cat /etc/msmtprc)
+-------------------------------------------
+
+Best regards
+The Nextcloud VM team
```

Comment:
Maybe change to NcVM..?

Reply:
That is your decision. Both are okay for me.